src/components/ProductList/ProductList.jsx

```diff
@@ -0,0 +1,87 @@
+import {React, useState, useEffect}  from 'react';
+import {ProductListContainer,SidebarWrapper,ProductCard,PaginationList} from './ProductList.styled';
+import {Loader} from './ProductList.styled';
+import mockProducts from '../../assets/mocks/en-us/product-categories.json';
+import {Card, CardText} from '../../components/Products/Products.styled';
+import mock3 from '../../assets/mocks/en-us/featured-products.json';
+
+export default function ProductList() {
+    const [selectedCategories, setSelectedCategories]= useState([]); //Para seleccionar los filtros
+    const [products, setProducts] = useState(mock3.results); //Para traer los productos
+    const [loading, setLoading] = useState(false); //Para el loader
+
+    useEffect(() => {
+        setLoading(true)
+        setTimeout(() => {
```

Comment:
alway that the code implements a setTimeout or setInterval, clear them to avoid memory leaks.
Also, what happen when any value of the state changes? the loader renders again?

 https://developer.mozilla.org/en-US/docs/Web/API/clearTimeout